dms/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Document Management System",
     "summary": """Document Management System for Odoo""",
-    "version": "12.0.4.0.7",
+    "version": "12.0.4.1.0",
```

Comment:
This changes some past behaviors, so I think it justifies a major bump.

```suggestion
    "version": "12.0.5.0.0",
```